x-pack/plugins/timelines/common/constants.ts

```diff
@@ -12,6 +12,12 @@ export const DEFAULT_NUMBER_FORMAT = 'format:number:defaultPattern';
 
 export const FILTER_OPEN: AlertStatus = 'open';
 export const FILTER_CLOSED: AlertStatus = 'closed';
+
+/**
+ * @deprecated
+ * TODO: Remove after `acknowledged` migration
```

Comment:
This can go with the issue above. Just don't want us to lose track of this and have a build up of dead code